xwiki-platform-core/xwiki-platform-localization/xwiki-platform-localization-rest/xwiki-platform-localization-rest-api/src/main/java/org/xwiki/localization/rest/TranslationResource.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.localization.rest;
+
+import java.util.List;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.QueryParam;
+
+import org.xwiki.localization.rest.model.jaxb.Translations;
+import org.xwiki.stability.Unstable;
+
+/**
+ * Exposes the wiki translations through REST.
+ *
+ * @version $Id$
+ * @since 13.3RC1
+ */
+@Path("/wikis/{wikiName}/localization/translations")
+@Unstable
+public interface TranslationResource
+{
+    /**
+     * Returns the raw translation values of the requested translation keys. In other words, the translations values
+     * without the parameters resolved.
+     * <p>
+     * The payload of a successful response is an {@link Translations} object containing a collection of {@link
+     * org.xwiki.localization.rest.model.jaxb.Translation}. Each translation is composed of a translation key (with the
+     * prefix concatenated), and the resolved translation raw value. If the translation key is not found, the raw value
+     * is {@code null}.
+     *
+     * @param wikiName the name of the wiki holding the translation keys
+     * @param locale the locale of the translation values, when {@code null} the current locale of the user is used
+     * @param prefix a common prefix, concatenated to each translation key before resolving their values (can be
+     *     {@code null} in which case nothing is concatenated)
+     * @param keys the translation keys to resolve. If no key is passed, and empty object is returned
+     * @return the list of resolved translations
+     */
+    @GET
+    Translations translations(@PathParam("wikiName") String wikiName, @QueryParam("locale") String locale,
```

Comment:
We use ``get`` prefix for GET requests everywhere in ``xwiki-platform-rest-api``. So it should be ``getTranslations``.